package/MDAnalysis/coordinates/memory.py

```diff
@@ -0,0 +1,257 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
```

Comment:
I'm not sure why this is showing up in this PR after we merged this in another PR?